### PR TITLE
added works and about section

### PR DESCRIPTION
**Pull request by r-ahmed2022**
updated the portfolio website with works and about section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -125,6 +128,172 @@ <h1 class="name-text">Hey There. <br>
...
+              <div id="div-1--left">
+              <img id="card-1-img" src="images/recenttitle.svg" alt="recent works">
+              </div>
+              <div id="div-1--right">
+              <p class="works-title-text">Multi-Post Stories</p>
+              <p id="card-1-text">A daily selection of privately personalized reads;
+                no accounts or sign-ups required. has been the industry's standard dummy text ever 
+                since the 1500s, 
+              when an unknown printer took a standard dummy text.</p>
+              <div id="skill-text">
```
Comment (danielufeli) [inline index.html:146]: - [ ] Nice work you have done on your My Recent Works section, kindly add some padding to the skills list so it aligns with the Figma design.

## Your Design
my-1

## Figma Design
my-2

File: index.html [hunk 2]
```diff
@@ -125,6 +128,172 @@ <h1 class="name-text">Hey There. <br>
...
+          </h2>
+         <p id="about-para">
+           Hello I’m a software developer! I can help you build a
+            product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project
+           you need coded, don’t hestiate to contact me.
+         </p>
+         <button class="div-1--right-btn">Get my Resume</button><br>
+         </div>
+      <!--  <div id="hr-line">
```
Comment (danielufeli) [inline index.html:261]: - [ ] Kindly remove all commented code to make you code more readable

File: index.html [hunk 3]
```diff
@@ -125,6 +128,172 @@ <h1 class="name-text">Hey There. <br>
...
+            product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project
+           you need coded, don’t hestiate to contact me.
+         </p>
+         <button class="div-1--right-btn">Get my Resume</button><br>
+         </div>
+      <!--  <div id="hr-line"> 
+         <hr>
+         </div>  -->
+        <div class="about-skills">
```
Comment (danielufeli) [inline index.html:264]: - [ ] Nice work you have done on your skills list, kindly align the lists so it aligns with the Figma Template.

## Your Design
f-11

## Figma Design
f-12

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @r-ahmed2022, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Highlights
Nice commits messages✔️
Descriptive PR ✔️
Clean Design ✔️
There is no linter error is present. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (r-ahmed2022) [reply index.html:146]: Added padding to list as advised

File: index.html (hunk 2 above)
Comment (r-ahmed2022) [reply index.html:261]: Removed commented code as advised

File: index.html (hunk 3 above)
Comment (r-ahmed2022) [reply index.html:264]: adding padding to the skills list in about section

**Comment by r-ahmed2022** [comment]
updated the works section with suggestions from the the Code reviewer

**Review by OLIPLICHE — APPROVED** [review]
Hi @r-ahmed2022 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**